src/script/cpp_api/s_item.h

```diff
@@ -35,13 +36,20 @@ class ScriptApiItem
 : virtual public ScriptApiBase
 {
 public:
+	/*
```

Comment:
Just return false if lua callback returns nil?

Reply:
I think not adding a second meaning to the return value is cleaner.

Reply:
Does it actually have the first meaning? :)
As far as I see it's used to skip setWieldedItem and some inventory change logic. Why don't make it like true - need to setWieldedItem(item) and handle the change and false - need to do nothing?

Reply:
The first meaning is "the callback did exist". You are correct that no existing code relies on it but I don't want to force the changes required on whoever needs both meanings in the future.